app/assets/stylesheets/publisher.scss

```diff
@@ -98,6 +98,12 @@
         line-height: $line-height-base !important;
       }
 
+      .twitter-typeahead {
+        position: absolute !important;
+
+        .tt-menu { margin-top: 0; }
```

Comment:
Ok. And in this case, it makes sense to enclose these rules into `#publisher_textarea_wrapper` do I ignore these remarks?